docs/how-to-run-ohmyglb.md

```diff
@@ -0,0 +1,105 @@
+# How to run Oh My GLB locally
+
+- [Overview](#overview)
+- [Environment prerequisites](#environment-prerequisites)
+- [Running project locally](#running-project-locally)
+- [See if everything is well installed](#see-if-everything-is-well-installed)
+- [See if everything running smoothly](#see-if-everything-running-smoothly)
+- [Cleaning](#cleaning)
+
+## Overview 
+The document helps contributors participating on GSLB to install local environment. 
+It is not intend to force you to strictly follow the document, but it can help you to 
+avoid troubles related setup local environment. 
+  
+
+![](./images/specialist2.jpg)
```

Comment:
They guy is not a vim fan and still using docker-compose - sorry, we can't merge it in :D 

Jokes aside, given that it is not a design doc - maybe upload it to e.g. https://imgur.com/ instead of committing to the repo.